core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -459,6 +461,21 @@ export class WorkflowActionService {
     this.executeStoreAndPropagateCommand(command, commandMessage);
   }
 
+  public addCommentBox(commentBox: CommentBox): void {
+    const currentHighlights = this.jointGraphWrapper.getCurrentHighlights();
+    const command: Command = {
+      modifiesWorkflow: true,
+      execute: () => {
+        this.jointGraphWrapper.unhighlightElements(currentHighlights);
+        this.jointGraphWrapper.setMultiSelectMode(false);
+        this.addCommentBoxInternal(commentBox);
+      },
+      undo: () => {
+        this.deleteCommentBoxInternal(commentBox.commentBoxID);
+      },
+    };
+    this.executeStoreAndPropagateCommand(command);
```

Comment:
It would be nice if this PR can make commenting work with shared editing, it should be a very easy change, you only need to follow what other actions (like `deleteOperator`) are doing.